crates/autopilot/src/domain/settlement/auction.rs

```diff
@@ -0,0 +1,22 @@
+//! Auction data related to the specific settlement.
+
+use {
+    crate::domain::{self},
+    std::collections::HashMap,
+};
+
+#[derive(Debug)]
+pub struct Auction {
+    pub id: domain::auction::Id,
+    /// Auction external prices
+    pub prices: domain::auction::Prices,
+    /// Fee policies for all orders in the auction. For some orders, there may
+    /// be no fee policies.
+    pub fee_policies: HashMap<domain::OrderUid, Vec<domain::fee::Policy>>,
```

Comment:
Should we call this orders then (and comment that it also contains the fee policies)?

Reply:
Yes, renamed 👍